packages/target-electron/src/windows/video-call.ts

```diff
@@ -132,6 +134,18 @@ const enum CallDirection {
 
 function openVideoCallWindow<T extends CallDirection>(
   accountId: number,
+  /**
+   * Whatever is known, either chatId or messageId, should be passed.
```

Comment:
```suggestion
   * Depending on the call direction we only know
   * the chat ID or the call message ID
```

Reply:
This doesn't have to be call direction though.